pkgs/development/interpreters/spidermonkey/78.nix

```diff
@@ -96,6 +95,15 @@ stdenv.mkDerivation rec {
 
   enableParallelBuilding = true;
 
+  postPatch = ''
+    # This patch is a manually applied fix of
+    #   https://bugzilla.mozilla.org/show_bug.cgi?id=1644600
+    # Once that bug is fixed, this can be removed.
+    # This is needed in, for example, `zeroad`.
+    sed -i "s/class SharedArrayRawBufferRefs {/class JS_PUBLIC_API SharedArrayRawBufferRefs {/g" \
```

Comment:
Can we use here substituteInPlace to make this simpler?

Reply:
updated